react/Menu.tsx

```diff
@@ -107,19 +125,19 @@ export default class Menu extends Component {
   /**
    * Convert the string params to a js object
    */
-  getParams = params => {
-    const json = {}
+  private getParams = (params: string | undefined): { [key: string]: string } => {
```

Comment:
```suggestion
  private getParams = (params?: string): { [key: string]: string } => {
```

Why not use optional parameter?